src/WebJobs.Extensions.SendGrid/Config/SendGridConfiguration.cs

```diff
@@ -68,28 +66,35 @@ public void Initialize(ExtensionConfigContext context)
                 throw new ArgumentNullException("context");
             }
 
-            INameResolver nameResolver = context.Config.NameResolver;
-            _defaultApiKey = nameResolver.Resolve(AzureWebJobsSendGridApiKeyName);
+            var metadata = new ConfigMetadata();
```

Comment:
This is pulling the config from Host.json. This is code that used to be in SendGridScriptBinding.Initialize.  The ConfigMetadata class matches the existing SendGrid config schema (which has slightly different naming than this config object)